repos/system_upgrade/el8toel9/actors/checkvdo/libraries/checkvdo.py

```diff
@@ -111,36 +111,36 @@ def _process_undetermined_conversion_devices(devices):
 
     unchecked = [x for x in devices if not x.check_failed]
     if unchecked:
-        no_vdo_devices = api.current_actor().get_no_vdo_devices_response()
-        if no_vdo_devices:
-            summary = ('User has asserted there are no VDO devices on the '
-                       'system in need of conversion to LVM management.')
-
-            reporting.create_report([
-                _report_title,
-                reporting.Summary(summary),
-                reporting.Severity(reporting.Severity.INFO),
-                reporting.Groups([reporting.Groups.SERVICES, reporting.Groups.DRIVERS]),
-                reporting.Groups([])
-            ])
-        elif no_vdo_devices is False:
-            summary = ('User has opted to inhibit upgrade in regard to '
-                       'potential VDO devices requiring conversion to LVM '
-                       'management.')
-            remedy_hint = ('Install the \'vdo\' package and re-run upgrade to '
-                           'check for VDO devices requiring conversion.')
-
-            reporting.create_report([
-                _report_title,
-                reporting.Summary(summary),
-                reporting.Severity(reporting.Severity.HIGH),
-                reporting.Groups([reporting.Groups.SERVICES, reporting.Groups.DRIVERS]),
-                reporting.Remediation(hint=remedy_hint),
-                reporting.Groups([reporting.Groups.INHIBITOR])
-            ])
+        summary = ('User has opted to inhibit upgrade in regard to '
+                   'potential VDO devices requiring conversion to LVM '
+                   'management.')
+        remedy_hint = ('Install the \'vdo\' package and re-run upgrade to '
+                       'check for VDO devices requiring conversion.')
+
+        reporting.create_report([
+            _report_title,
+            reporting.Summary(summary),
+            reporting.Severity(reporting.Severity.HIGH),
+            reporting.Groups([reporting.Groups.SERVICES, reporting.Groups.DRIVERS]),
+            reporting.Remediation(hint=remedy_hint),
+            reporting.Groups([reporting.Groups.INHIBITOR])
+        ])
 
 
 def check_vdo(conversion_info):
-    _process_pre_conversion_vdos(conversion_info.pre_conversion)
-    _process_post_conversion_vdos(conversion_info.post_conversion)
-    _process_undetermined_conversion_devices(conversion_info.undetermined_conversion)
+    all_vdo_converted = api.current_actor().get_all_vdo_converted_response()
+    if all_vdo_converted:
+        summary = ('User has asserted all VDO devices on the system have been '
+                   'successfully converted to LVM management.')
+
+        reporting.create_report([
+            _report_title,
+            reporting.Summary(summary),
+            reporting.Severity(reporting.Severity.INFO),
+            reporting.Groups([reporting.Groups.SERVICES, reporting.Groups.DRIVERS]),
+            reporting.Groups([])
```

Comment:
Just noticed this line is redundant, however, it is not blocking this patch to be merged. Removing the line is up to you.
```suggestion
```

Reply:
Petr addressed this in his push.